include/types.h

```diff
@@ -1543,7 +1559,8 @@ typedef struct debugfile_ctx
 {
   bool enabled;
 
-  FILE *fp;
+//  FILE *fp;
```

Comment:
obsolete comment

Reply:
What I mean is, please remove the comment itself. It makes it easier to read.

```
//  FILE *fp;
```

Same applies for all comments which you are using in all other files.

Reply:
ok